README.md

```diff
@@ -17,6 +17,16 @@ To install or update to latest version of CLI use following script:
 curl -o- https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/scripts/install.sh | bash 
 ```
 
+## RHOAS Container Image
+
+Container image for RHOAS can be found in [redhat-developer/app-services-tools](https://github.com/redhat-developer/app-services-tools). It contains a number of CLIs that can be executed as container.
```

Comment:
```suggestion
RHOAS CLI can be also used as container image. This can be useful when using CLI in OpenShift environment, automation and testing.. Image contains also openshift cli and kafkacat binaries. For more information please refer to [redhat-developer/app-services-tools](https://github.com/redhat-developer/app-services-tools) github repository
```